modules/user/user.repository.js

```diff
@@ -59,6 +97,7 @@ class UserRepository {
     }
 
     async saveNewUser(newUser) {
+        newUser.password = await bcrypt.hash(newUser.password, saltRounds);
```

Comment:
this line should on service layer too. repository shouldn't contain any logic. Only read/write/delete/update data from DB

- [ ] move hashing logic to service layer